addons/mail/static/src/discuss/call/public_web/discuss_sidebar_call_participants.xml

```diff
@@ -2,11 +2,18 @@
 <templates xml:space="preserve">
     <t t-name="mail.DiscussSidebarCallParticipants">
         <t t-if="sessions.length gt 0">
-            <Dropdown t-if="compact" state="floating" position="'right-middle'" manual="true" menuClass="'py-0'">
+            <Dropdown t-if="compact" state="floating" position="'right-start'" manual="true" menuClass="'py-0'">
                 <t t-call="mail.DiscussSidebarCallParticipants.main"/>
                 <t t-set-slot="content">
                     <div t-ref="floating">
-                        <DiscussSidebarCallParticipants thread="props.thread" compact="false"/>
+                        <div class="d-flex align-items-center px-2 py-1 gap-1" t-foreach="sessions" t-as="session" t-key="session.localId">
+                            <t t-set="persona" t-value="session.channelMember.persona"/>
+                            <img t-att-src="persona.avatarUrl" class="rounded-circle" style="width:24px;height:24px"/>
+                            <div class="d-flex align-items-baseline">
+                                <span class="mx-1 text-truncate fw-bolder smaller" t-esc="persona.name"/>
+                                <t t-call="mail.DiscussSidebarCallParticipants.statuses"/>
+                            </div>
+                        </div>
```

Comment:
Should display isTalking in floating menu compact mode:
<img width="1279" alt="Screenshot 2024-09-27 at 15 59 18 (3)" src="https://github.com/user-attachments/assets/d14f47eb-f32e-4666-b18d-620cf54fa38c">